### ISSUE TITLE
Installer failing in an UpdateManager method on first install 

### ISSUE THREAD

**Issue by floydpink** [Squirrel/OldSquirrelForWindows#247]
I could follow the documentation in the [wiki](https://github.com/Squirrel/Squirrel.Windows/wiki) as well as the instructions to use the Shimmer package from [Bradley Grainger's gist](https://gist.github.com/bgrainger/7287703) to successfully create the windows installer for [my mono-based multi-platform app](https://github.com/flickr-downloadr/flickr-downloadr-gtk/tree/squirrel).

But when I run the setup.exe and proceed with the installation I get an error and the log file content is as below:

``` plain
[WARN][2014-04-14T04:33:13] POCOObservableForProperty: Shimmer.WiXUi.Views.WelcomeView is a POCO type and won't send change notifications, WhenAny will only return a single value!
[INFO][2014-04-14T04:33:21] WixUiBootstrapper: PlanCompleteObs: got status: '0'
[WARN][2014-04-14T04:33:21] POCOObservableForProperty: Shimmer.WiXUi.Views.InstallingView is a POCO type and won't send change notifications, WhenAny will only return a single value!
[WARN][2014-04-14T04:33:21] InstallManager: Updating using deltas has failed: System.InvalidOperationException: AwaitableAsyncSubject<T>.GetResult() is rethrowing an inner exception ---> System.InvalidOperationException: Sequence contains more than one matching element
   at System.Linq.Enumerable.SingleOrDefault[TSource](IEnumerable`1 source, Func`2 predicate)
   at Shimmer.Client.UpdateManager.findCurrentVersion(IEnumerable`1 localReleases)
   at Shimmer.Client.UpdateManager.determineUpdateInfo(IEnumerable`1 localReleases, IEnumerable`1 remoteReleases, Boolean ignoreDeltaUpdates)
   at Shimmer.Client.UpdateManager.<>c__DisplayClass4.<checkForUpdate>b__3(IEnumerable`1 releases)
   at System.Reactive.Linq.Observαble.SelectMany`2._.OnNext(TSource value)
   --- End of inner exception stack trace ---
   at ReactiveUIMicro.AwaitableAsyncSubject`1.GetResult()
   at Shimmer.Client.InstallManager.<executeInstall>d__9.MoveNext()
[ERROR][2014-04-14T04:33:21] WixUiBootstrapper: Something unexpected happened: System.InvalidOperationException: AwaitableAsyncSubject<T>.GetResult() is rethrowing an inner exception ---> System.InvalidOperationException: Sequence contains more than one matching element
   at System.Linq.Enumerable.SingleOrDefault[TSource](IEnumerable`1 source, Func`2 predicate)
   at Shimmer.Client.UpdateManager.findCurrentVersion(IEnumerable`1 localReleases)
   at Shimmer.Client.UpdateManager.determineUpdateInfo(IEnumerable`1 localReleases, IEnumerable`1 remoteReleases, Boolean ignoreDeltaUpdates)
   at Shimmer.Client.UpdateManager.<>c__DisplayClass4.<checkForUpdate>b__3(IEnumerable`1 releases)
   at System.Reactive.Linq.Observαble.SelectMany`2._.OnNext(TSource value)
   --- End of inner exception stack trace ---
   at ReactiveUIMicro.AwaitableAsyncSubject`1.GetResult()
   at Shimmer.Client.InstallManager.<executeInstall>d__9.MoveNext()
[WARN][2014-04-14T04:33:21] POCOObservableForProperty: Shimmer.WiXUi.Views.ErrorView is a POCO type and won't send change notifications, WhenAny will only return a single value!
```

Could someone please help in figuring out how I can resolve this?

Thanks.

**Comment by bgrainger** [comment]
It appears that [`findCurrentVersion`](https://github.com/Squirrel/Squirrel.Windows/blob/master/src/Squirrel.Client/UpdateManager.cs#L403) will fail if there are two entries (in the list of releases) that have the same version. It's possible this could be caused by a malformed `RELEASES` file on your update server. What's the URL you're passing to the `UpdateManager` constructor in your code?

**Comment by floydpink** [comment]
Thanks for helping out, @bgrainger. 

I do not have an update server (yet) and I am not using the `UpdateManager` directly. Here are the things I did:
1. Installed Shimmer package (which in turn installed Shimmer.Client and Shimmer.Core)
2. Updated the projects to include the `BuildPackage` element.
3. Invoked the `Create-Release` powershell script from [the build script](https://github.com/flickr-downloadr/flickr-downloadr-gtk/blob/squirrel/build/create-release-win.ps1)

This did give me the installer, the `RELEASES` file and accompanying `.nupkg` files (as can be seen [here](https://github.com/flickr-downloadr/flickr-downloadr-gtk/tree/squirrel/dist/win)), but the installer itself was giving me the error pasted above.
